public/app/features/dashboard/components/AddPanelButton/AddPanelButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { css, cx } from '@emotion/css';
+import React, { useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Dropdown, Button, useTheme2, Icon } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+import { DashboardModel } from 'app/features/dashboard/state';
+
+import { AddPanelMenu } from './AddPanelMenu';
+
+interface Props {
+  dashboard: DashboardModel;
+}
+
+export const AddPanelButton = ({ dashboard }: Props) => {
+  const styles = getStyles(useTheme2());
+  const [addMenuOpen, setAddMenuOpen] = useState(false);
```

Comment:
Nit: maybe `isAddMenuOpen`? as this is a boolean :)

Reply:
+1